### PR TITLE
Gareth/feed fixes

### PR DESCRIPTION
**Pull request by pcGarethJames**
- Tweaks to feed ajax & js processing of return for feeds & signup
- Removed disabled feed button when feeds running, so that feeds can be re-run easily
- Added extra status resets to feed submission, so that feed status gets reset for stuck feeds
- Improvements to feed status display so that current running feed is shown correctly, and also a finished feed in a run is correctly flagged as complete while other feeds are still running

### CODE REVIEW

File: classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php [hunk 1]
```diff
@@ -132,14 +135,34 @@ protected function sendFeeds(array $feedTypes)
         }
 
         if (in_array(Feed::FEED_TYPE_USER, $feedTypes)) {
+            $feedTypes = $this->removeFeedFromState($feedTypes, Feed::FEED_TYPE_USER);
             $userFeed = UserFeedRunner::getInstance();
             $userFeed->runFeed();
+            $feedTypes = $this->removeFeedFromState($feedTypes, Feed::FEED_TYPE_BRAND);
```
Comment (PeterBrooksbank) [inline classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php]: Suspect this should be in the brand feed section?

File: classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php [hunk 2]
```diff
@@ -132,14 +135,34 @@ protected function sendFeeds(array $feedTypes)
         }
 
         if (in_array(Feed::FEED_TYPE_USER, $feedTypes)) {
+            $feedTypes = $this->removeFeedFromState($feedTypes, Feed::FEED_TYPE_USER);
```
Comment (PeterBrooksbank) [inline classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php]: You've placed the others at the bottom of each block, any difference here?

**Review by PeterBrooksbank — CHANGES_REQUESTED** [review]
Couple of comments

File: classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php (hunk 2 above)
Comment (pcGarethJames) [reply classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php]: C&P error, should be at the bottom, fixed now :)

File: classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php (hunk 1 above)
Comment (pcGarethJames) [reply classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php]: yup, it should be , thanks for spotting, fixed now :)

**Review by PeterBrooksbank — APPROVED** [review]
LGTM

**Pull request merged**